scripts/sdk_process/patch_models.py

```diff
@@ -232,6 +232,7 @@ def find_autorest_generated_folder(module_prefix="azure.mgmt"):
         'azure.mgmt.network',
         'azure.mgmt.monitor',
         'azure-mgmt-loganalytics'
+        'azure-mgmt.rdbms'
```

Comment:
```suggestion
        'azure.mgmt.rdbms'
```
@jsntcy to check

Reply:
Why use `-` between `azure` and `mgmt`, but `.` between `mgmt` and `rdbms`? I'm little confused